scripts/security_data_copy.py

```diff
@@ -0,0 +1,17 @@
+from distutils.dir_util import copy_tree
+from argparse import ArgumentParser
+
+parser = ArgumentParser()
+parser.add_argument('--source_path', type=str)
+parser.add_argument('--security_destination', type=str)
+parser.add_argument('--experiment', type=str)
+
+destination_path_name = '{}/experiment_{}'
```

Comment:
Could you name the params in this kind of string format? 
For example:
```suggestion
destination_path_name = '{destination}/experiment_{experiment}'
```
And then, when using this string, you could do:
```
destination_path_name.format(destination=security_destination, experiment=experiment)
```

I think it improves the readability and put this script into the pattern in which others python script we use are in.